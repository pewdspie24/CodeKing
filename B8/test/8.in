17
46
978077942 701799457 15608377 655406194 959038551 443776830 714537931 111918367 694882256 542187933 863915454 174256949 121576710 4888302 646871797 546076715 606788037 440892194 949530091 240315204 139603586 538938206 213925257 727854600 933055080 496286631 551995540 400469713 110360855 839066875 582273749 847785748 884103860 253958486 132969826 683185911 334238459 172822485 546368386 101950754 774430728 634566470 322160341 647498869 239131031 621201802 
24
246409698 377488044 887988804 758185249 791443095 631208826 908922214 249005708 727450776 510964268 772850537 875014257 485515973 60900299 708680904 402152057 870139627 837160897 354173344 510168207 164232493 182078178 114366162 864131131 
39
508257103 536207474 208221673 237183136 877280624 685941698 515678666 65493535 204841915 583777630 276410108 66838556 876082962 194614425 445711348 43995525 873508882 831613958 404167272 286867667 193295872 946470722 514608286 166470045 763361021 19965182 13428371 849074438 564286916 101740355 528860096 189913106 942698393 922253552 108037006 333384633 269652416 972898783 219090669 
41
541684328 213290292 323159343 582677578 181507406 951442851 682577589 617579842 101440331 103536631 496745036 146128125 420304492 176969083 872954677 929931969 698376568 698534318 897415658 620892416 598582819 395319980 329102901 130082405 132445648 993011734 637827808 696496202 816409263 953384980 456334682 692712634 507920032 391563207 737298631 577821522 519658549 619572800 41403787 558648483 633331693 
34
818810639 660893524 636855214 157072175 178541214 14802350 19612709 978975782 922339563 825066814 270590616 219809495 826483622 43783230 790239509 161523886 109271133 299483649 169616341 714591071 728050853 602922048 864216096 791835919 112520410 293355569 637372703 606408749 67596154 750923695 331301172 263981756 187421454 110909041 
37
45559392 8380061 775698837 906242299 503979985 25000932 903784989 3946197 32983078 208739041 205425311 482355978 94436545 695501571 281503602 278622261 965581671 585952751 952171660 239138535 51401747 544838391 552224057 915338723 418903447 834916803 344068426 127734541 246489446 344239558 83566440 245334656 273544245 112914382 792671903 3283343 226859095 
27
498328268 315886056 115907018 448636494 543013467 46521377 225959172 153533985 975111147 154169956 232374640 444882766 250493671 597572618 32400532 752058705 950536184 109305468 933330968 983084024 529559559 875879004 931966725 425117009 161886057 514015881 204528688 
21
83493105 450291632 987201308 988797970 738089694 540301579 593899111 843060244 794462725 889140334 258307508 718768839 596911747 616742381 485901692 524507783 802284606 135602941 439681124 214462700 883179590 
48
701758286 287890801 527670700 505829517 495544102 486498595 957880637 738960336 951199467 208566142 859771942 218917479 951337790 182009472 783892276 89615284 563418353 355675536 469966348 782355225 183016523 613215698 120222146 927446815 203485919 375837315 842431590 790582836 151632972 733150659 367174213 516138404 464605037 629717796 724833564 513798929 434386390 165542754 405084319 15413689 223431913 608076475 163613665 209289268 99596388 104812586 169012889 762664938 
41
271518750 907040740 569613235 900993494 134909700 752212585 966912476 800021920 931969551 744987067 617091693 150265561 469634295 274692148 505041328 296825676 294693414 174569480 370096919 456291592 903020198 296653176 251801757 531956442 584988538 885337978 563174386 324104224 998772219 121911816 380424930 685826856 556841321 179869905 399772117 676610417 801894787 619657959 381827671 342997538 753413981 
46
105732932 546312632 40810238 943140182 319283117 277868413 351056094 859611109 20343652 924547916 805941170 65379156 797042922 131029991 238622393 576722526 565001544 767015692 857373979 874108152 966789715 236715292 864663837 948521051 187982887 535506206 792733954 977822800 733107810 315876874 342102618 968438545 549933172 432171811 972974558 327703229 859254055 484086710 547606849 177371867 216476129 475353098 66098529 763985456 791710912 664313952 
21
993022083 146336955 802575270 551493895 77683134 122876881 246343801 816835394 228568910 293285456 266465661 304987284 109932844 321771125 805012899 76843495 351209568 251479215 222129607 398107259 202775347 
43
812601772 293520536 291949842 800386167 497605868 135147709 84279646 172858880 278347641 539021556 592467595 37593938 73337640 585540254 132788637 165122627 786464303 491435445 246074027 480798017 937463505 708254459 973769820 989943412 195819428 500310840 237493365 374384869 62961065 743336482 175955948 630962007 536824198 549833531 465693575 782804833 836660016 401409090 39512473 511840250 297215279 604589306 407134813 
26
271757766 276621985 768748068 280446506 897467763 391238075 373962606 767093916 364964530 765576336 820749428 550560844 44170218 255733947 185062498 216644493 955482078 313790682 572961195 550298294 265952823 704107431 462415745 114610496 833279661 718194433 
29
979654353 66226633 185004356 27448222 275369015 92220186 863205177 520553549 147342970 951698914 8031088 846734242 179785877 921593986 143630373 387700515 418585650 113637598 919241451 524678460 201525008 697826418 775904633 161054130 722702222 214198194 357713348 69806914 946736565 
20
491167718 14630855 442212795 399763727 447738081 592177261 483553348 510877267 931011076 600072120 291253718 34594656 233038630 739164464 86141061 884017147 978461603 241631958 187303303 814504952 
37
405489984 979267390 404800984 700791469 785224415 56791118 955126854 124065521 152305125 886409885 596033753 3151781 990408045 954510747 892715078 518531075 922320620 639430253 105355861 155988411 586134547 196865471 80864733 440994247 604293913 654616666 153613129 866969618 55939093 519976152 563395337 492142398 213842840 432263973 952387972 822406728 354235121 
