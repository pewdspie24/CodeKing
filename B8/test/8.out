Bi
Bi
Bi
Lema
Bi
Bi
Lema
Lema
Lema
Bi
Bi
Bi
Lema
Bi
Bi
Lema
Bi
